include/z64player.h

```diff
@@ -256,203 +255,200 @@ typedef struct struct_80122744_arg1 {
     /* 0x4 */ Vec3s* unk_04;
 } struct_80122744_arg1; // size = 0x8
 
-// 
-#define PLAYER_STATE1_1          (1 << 0)
-// 
-#define PLAYER_STATE1_2          (1 << 1)
-// 
-#define PLAYER_STATE1_4          (1 << 2)
-// 
-#define PLAYER_STATE1_8          (1 << 3)
-// 
-#define PLAYER_STATE1_10         (1 << 4)
-// 
-#define PLAYER_STATE1_20         (1 << 5)
-// 
-#define PLAYER_STATE1_40         (1 << 6)
-// 
-#define PLAYER_STATE1_80         (1 << 7)
-// 
-#define PLAYER_STATE1_100        (1 << 8)
-// 
-#define PLAYER_STATE1_200        (1 << 9)
-// 
-#define PLAYER_STATE1_400        (1 << 10)
-// 
-#define PLAYER_STATE1_800        (1 << 11)
-// 
-#define PLAYER_STATE1_1000       (1 << 12)
-// 
-#define PLAYER_STATE1_2000       (1 << 13)
-// 
-#define PLAYER_STATE1_4000       (1 << 14)
-// 
-#define PLAYER_STATE1_8000       (1 << 15)
-// 
-#define PLAYER_STATE1_10000      (1 << 16)
-// 
-#define PLAYER_STATE1_20000      (1 << 17)
-// 
-#define PLAYER_STATE1_40000      (1 << 18)
-// 
-#define PLAYER_STATE1_80000      (1 << 19)
-// 
-#define PLAYER_STATE1_100000     (1 << 20)
-// 
-#define PLAYER_STATE1_200000     (1 << 21)
-// 
-#define PLAYER_STATE1_400000     (1 << 22)
-// 
-#define PLAYER_STATE1_800000     (1 << 23)
-// 
-#define PLAYER_STATE1_1000000    (1 << 24)
-// 
-#define PLAYER_STATE1_2000000    (1 << 25)
-// 
-#define PLAYER_STATE1_4000000    (1 << 26)
+//
+#define PLAYER_STATE1_1 (1 << 0)
```

Comment:
Please don't format the headers, we prefer to keep them more consistent than the formatter manages.

Reply:
My bad, vscode set to autoformat on save. After undoing the name change, I had no intent for any modifications in this file to remain.